hack/tools/install_oc.sh

```diff
@@ -17,9 +16,9 @@ target_os="$os"
 if [[ "$target_os" == "darwin" ]]; then
     target_os="mac"
 fi
-curl -L -O https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${version}/openshift-client-${target_os}-${version}.tar.gz
-trap "rm openshift-client-${target_os}-${version}.tar.gz" err exit
+curl -L -O https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_VERSION}/openshift-client-${target_os}-${OC_VERSION}.tar.gz
```

Comment:
you don't check oc's version, and don't print what is checked and installed version